adapters/defaultLogger/defaultLogger.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package defaultLogger
+
+import (
+	"errors"
+	"os"
+	"text/template"
+
+	"istio.io/mixer"
+	"istio.io/mixer/adapters"
+)
+
+const (
+	name = "istio.io/mixer/loggers/defaultLogger"
+	desc = "Creates access log entries from Report() requests and flushes to os.Stdout"
+	tmpl = "defaultLogger"
+)
+
+type (
+	builder struct{}
+	adapter struct {
+		t *template.Template
+	}
+
+	// Config represents the adapter configuration for the default logger.
+	Config struct {
+		// Template is used to pass a golang text/template string to the
+		// adapter for translation of fact values into a log entry.
```

Comment:
This is the bit I am trying to flesh out. If we remove LogEntry from the API surface (which it sounded like we had consensus for), I don't see how we look up Descriptors, etc., ahead of adapters. How would we even know what descriptor to look up?

In a previous iteration of this PR, I had defined a `mixer.LogEntry` and a `Logger` interface that exposed a `Log([]mixer.LogEntry)`. But that requires some sort of agreed upon schema for logs across logging adapters.

What we are left with, as I understand it, is a set of `facts` or (`FactValues` in the code) and some config (in this case, the template format string) for the logging adapter.  Am I missing something?

The transformation from a bag of facts to a log line seems adapter-specific (maybe your default logging wants a different template than your stackdriver logger), but maybe it is OK to move it out of the adapters themselves.

@mandarjog any thoughts here?